docs/synthetics/index.md

```diff
@@ -14,7 +14,7 @@ Sematext Synthetics provides the ability to actively monitor APIs, Web URLs, web
 
 You start by creating a Synthetics [App](/guide/app-guide/). Under the App, you can create an HTTP or Browser monitor. HTTP monitors can be used to monitor any HTTP endpoints like HTTP APIs or Web URLs. An HTTP monitor sends an HTTP request to the specified URL with configured request settings and records the timings and response details. Browser monitors can be used to monitor websites and user journeys by writing a script that simulates user actions. The Browser monitor script executes in a browser and the performance is recorded. While creating the monitor you can specify a list of conditions that decides the result of the run.
 
-No agent installation is required and no need to change your application code. The monitors will be periodically scheduled to run from specified locations in an isolated environment. No agent installation is required. For example, a monitor scheduled to run every 15 minutes from 3 locations will run 12 times in an hour (and 8640 times a month). The monitor will run on average every 5 minutes in each 15 minutes interval.
+No agent installation is required and there is no need to change your application code. The monitors will be periodically scheduled to run from specified locations in an isolated environment. No agent installation is required. For example, a monitor scheduled to run every 15 minutes from 3 locations will run 12 times in an hour (and 8640 times a month). The monitor will run on average every 5 minutes in each 15 minutes interval.
```

Comment:
"No agent installation is required." is redundant, no?

Reply:
You mean it is redundant with `no need to change your application code`? By no need to change application code, I meant not instrumentation is needed, like APM. By no agent installation, we mean something like our sematext-agent

Reply:
I'm referring to:
"No agent installation is required and there is no....."
and then
"No agent installation is required."

Reply:
Aha. Sorry. Missed the second sentence. Will remove it.